mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -50,123 +54,311 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
 
+    private int batchSize=64;
```

Comment:
No rational, just some arbitrary number. We might make it configurable via
the datasource maybe?


On Fri, May 6, 2016 at 11:22 AM, Naveen Malik <notifications@github.com>
wrote:

> In mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/259#discussion_r62360942>
> :
>
> > @@ -50,123 +54,311 @@
> >
> >      private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
> >
> > +    private int batchSize=64;
>
> Rational behind 64? It's hard coded, is it worth exposing and making
> configurable via puppet? If yes we could create an issue to fix do that
> later.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/259/files/a9fc8b0e5beacdb201ecc3189e3ce88ccf38ad1f#r62360942>
>